lib/Backends/CPU/CPUBackend.h

```diff
@@ -67,6 +67,14 @@ class CPUBackend : public LLVMBackend {
   virtual bool
   canDoIndexTypeDemotion(ElemKind fromTy, ElemKind toTy,
                          PrecisionConfiguration &precConfig) const override;
+
+  /// \returns an array of raw objects which are statically allocated and
```

Comment:
This is an override and probably just copies the comment from a parent method? If so, let's not repeat it here.

Reply:
Removed the comment. Done.